library/pk_wrap.c

```diff
@@ -1009,7 +1009,7 @@ static int ecdsa_sign_wrap(void *ctx, mbedtls_md_type_t md_alg,
 #endif /* MBEDTLS_USE_PSA_CRYPTO */
 #endif /* MBEDTLS_PK_CAN_ECDSA_SIGN */
 
-#if defined(MBEDTLS_PK_CAN_ECDSA_SOME)
+#if defined(MBEDTLS_ECDSA_C)
```

Comment:
Minor: I think putting the two `#if`s on one line with `&&` would be slightly more readable, but that's not worth doing another round of CI+review just for this.